dag/archive/energy.yml

```diff
@@ -169,3 +169,14 @@ steps:
     - data://garden/eia/2022-07-27/energy_consumption
     - data://garden/ggdc/2020-10-01/ggdc_maddison
     - data://garden/owid/latest/key_indicators
+  #
+  # Energy - OWID Energy dataset (2023).
+  #
+  data://garden/energy/2023-02-20/owid_energy:
+    - data://garden/bp/2023-02-20/energy_mix
+    - data://garden/energy/2023-02-20/fossil_fuel_production
+    - data://garden/energy/2023-02-20/primary_energy_consumption
+    - data://garden/energy/2023-02-20/electricity_mix
+    - data://garden/demography/2022-12-08/population
```

Comment:
Is this a new dataset? If so, should probably use the latest population dataset: `data://garden/demography/2023-03-31/population`

Reply:
Same for other files, where `2022-12-08` version is used, too.